actions/views/rhs.js

```diff
@@ -280,6 +280,34 @@ export function selectPost(post) {
     };
 }
 
+export function setShowPreviewOnCreateComment(showPreview) {
+    return {
+        type: ActionTypes.SET_SHOW_PREVIEW_ON_CREATE_COMMENT,
+        showPreview,
+    };
+}
+
+export function setShowPreviewOnCreatePost(showPreview) {
+    return {
+        type: ActionTypes.SET_SHOW_PREVIEW_ON__CREATE_POST,
```

Comment:
Could you remove the doubled up underscore here?
```suggestion
        type: ActionTypes.SET_SHOW_PREVIEW_ON_CREATE_POST,
```